Update Makefile and README to cross compile for x86_64 to arm device (like Raspberry Pi)

Keuronde opened the pull request:
Update usage of libhid-dev to libusb-dev in readme

Remove 'strip' command in the makefile, couldn't make it work for arm.

Executable tested OK on Raspberry Pi Zero

Commit:
Update Makefile and README to cross compile for x86_64 to arm device
Update usage of libhid-dev to libusb-dev in readme

AdamLaurie reviewed (changes requested):
thanks for the PR and apologies for the very slow response!
please take DEBUG back out
instead of removing STRIP i suggest you override its command for your targets - maybe set to echo some message?
please remove binaries - i will compile and add them

Commit:
Taking into account AdamLaurie advices :
- Remove Debug flag
- Keep strip command for x86 (32bits and 64bits) architecture
- Add some @echo command to explain the build target

In addition, using gcc-10 instead of gcc-8 since in debian 12, this is the default version.

Keuronde commented:
Hey, thanks for answering this pull request !
I have updated the Makefile following your advises. Please tell me if it's OK now !
